genericjpa/db/src/main/java/org/hibernate/search/annotations/UpdateInfo.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.annotations;
+
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * @author Martin Braun
+ * @hsearch.experimental
+ */
+@Target({FIELD, METHOD, TYPE})
+@Retention(RUNTIME)
+@Repeatable(UpdateInfos.class)
+public @interface UpdateInfo {
```

Comment:
@gunnarmorling 

I think I have an alternative that can make us both happy:

For the sake of _this_ pullrequest we keep the extra annotations for Hibernate Search when used with Hibernate ORM. I found out that it should be possible to build a "EventModelParser" (that's what I named the interface that gathers the information about the tables) that uses information from Hibernate ORM's persisting system to construct the needed meta-objects (and falls back to the annotations if needed?).

For me this is actually quite hard as I don't know Hibernate ORM's code-base well at all and I will need some help with that.

/CC @Sanne